### ISSUE TITLE
Image URL

### ISSUE THREAD

**Issue by coultergeist** [ActoKids/AD440_W19_CloudPracticum#96]
For this final sprint, handle images not in the db but via URL. The demo should show a thumbnail of the image on the event list, as well as a full image on the Event Info page. See also the [Demo Issue](https://github.com/ActoKids/AD440_W19_CloudPracticum/issues/91) for more info. Once assigned, annotate your estimated time. Then, keep regular updates on your issue. Comment your approach, tutorials, articles, updates, links, photos, etc. Comment often. When closing your issue include the following in your final comment.

Time estimated:
Time spent:
Pull Request link:
Wiki link:
Deployed Code link:
Testers:
What I tested:

**Comment by zphupro** [comment]
i am going to show image  in each of list event when they created event and when they click on event it will show the picture 
Time estimated: 8

**Comment by zphupro** [comment]
Time estimated: 8
Time spent: 6
Pull Request link:  https://github.com/ActoKids/web-ui/pull/43
Wiki link: https://github.com/ActoKids/web-ui/wiki/Image-URL-for-Event-list-and-Event-details
Deployed Code link: https://2edusite.com/dashboard
Testers: erikNeville: Images are loading fine into EventList and EventSummary. Might change around size and location of them, but they are working.
  hkhual : Maybe need to change the image sizes, other than that, It looks good.

What I tested: https://github.com/ActoKids/web-ui/pull/37 I tested for Erik he makes change a lot for the branch like;  moved the Link around each event, added some comments to help understand and more. I test on my local machine and it works perfect .no problem
